packages/shared/styles/components/organisms/SfBottomNavigation.scss

```diff
@@ -4,55 +4,43 @@
   position: fixed;
   bottom: 0;
   left: 0;
-  z-index: var(--bottom-navigation-z-index, 1);
+  z-index: 1;
   display: flex;
   justify-content: space-around;
   align-items: flex-end;
   width: 100%;
-  height: var(--bottom-navigation-height, 3.75rem);
-  padding: var(--bottom-navigation-padding, 0 var(--spacer-sm));
-  background: var(--bottom-navigation-background, var(--c-white));
-  box-shadow: var(
-    --bottom-navigation-box-shadow,
-    0px -2px 10px rgba(var(--_c-gray-secondary-base), 0.15)
-  );
+  height: 3.75rem;
+  background: var(--c-white);
+  box-shadow: 0px -2px 10px rgba(var(--_c-gray-secondary-base), 0.15);
```

Comment:
shouldn't we have global css vars for box shadow and apply it here?

Reply:
We should. Can you open a PR for global css variables? :)

Reply:
@mayashavin yeah, i'll give a try